### ISSUE TITLE
Type declaration not found

### ISSUE THREAD

**Issue by deasems** [GoogleChrome/web-vitals#21]
Importing this package in a typescript project results in the error:

> Could not find a declaration file for module 'web-vitals'. '<snip>/node_modules/web-vitals/dist/web-vitals.min.js' implicitly has an 'any' type.
>   Try `npm install @types/web-vitals` if it exists or add a new declaration (.d.ts) file containing `declare module 'web-vitals';`ts(7016)

I see the lib is written in typescript so I'm assuming this is a misconfiguration of the build, not that types are hosted in an types package

**Comment by deasems** [comment]
Explicitly importing the non-minified file works around this issue:
```ts
import { getCLS, getFCP, getFID, getLCP, getTTFB } from 'web-vitals/dist/web-vitals'
```

**Comment by deasems** [comment]
It would also be useful to be able to import the `Metric` and `ReportHandler` types

**Comment by philipwalton** [comment]
> I see the lib is written in typescript so I'm assuming this is a misconfiguration of the build, not that types are hosted in an types package

Yep, sorry, it looks like pointing `pkg.main` at the minified version is preventing VS Code from finding the right type definitions. I think adding `web-vitals.min.d.ts` should do the trick. I'll get it fixed in the next release (either today or tomorrow).

**Comment by philipwalton** [comment]
> It would also be useful to be able to import the `Metric` and `ReportHandler` types

Ahh, good call, I'll make sure the main export includes everything in `/dist/types.d.ts`.

**Comment by philipwalton** [comment]
@deasems I just released `v0.2.1`, which should fix this. Please give it a try and let me know if you're still having issues.

**Comment by deasems** [comment]
Updated and import's working as expected, thanks for the quick turnaround @philipwalton
